src/board/mod.rs

```diff
@@ -243,19 +257,90 @@ impl Board {
             let color = self.next_player();
             self.vacant
                 .iter()
-                .map(|coord| Play(color.clone(), coord.col, coord.row))
+                .map(|coord| Play(color, coord.col, coord.row))
                 .filter(|m| self.is_legal(*m).is_ok())
                 .collect()
         }
     }
-
```

Comment:
The idea here is to pick a random legal move and the use it if it satisfies the eye check, right? I'd prefer to have this code inside of playout. Just put it there, I'm in the process of writing tests for that part anyway.